stdlib/universe/derivative_test.go

```diff
@@ -1013,13 +1004,17 @@ func TestDerivative_Process(t *testing.T) {
 	for _, tc := range testCases {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
-			executetest.ProcessTestHelper(
+			executetest.ProcessTestHelper2(
```

Comment:
Yes there definitely would be a better way to name this. The primary reason was because I just needed a way to test one function when I was first starting to refactor things and this was easy. It might be good to go through these and refactor all of the calls instead or rename this.